.codespellrc

```diff
@@ -0,0 +1,6 @@
+[codespell]
+skip = .git,*.pdf,*.svg,yarn.lock,*.min.js,locales
```

Comment:
oh does this mean it will run on js, ts, jsx, and tsx files too? How powerful is it to not mess around with real js code and only fix typos in the function documents

Reply:
well, you can see how well it did so far ;)

indeed JS and alike are typically trickier but not entirely hopeless -- worse comes to worse you can disable for those if starts bothering.  codespell doesn't do any programming language specific parsing -- so can't tell code from documentation. Just extracts words and finds typos in the words.

Reply:
Great explanation thanks again!